release_draft.md

```diff
@@ -0,0 +1,299 @@
+
+### Prerequisites
+
+To perform a new release, a small number of configurations and credentials need to be made on the developer's system before triggering the release.
+If a permanent operative system is used for releasing, these installation steps only need to be executed once.
+
+#### Team and development checks
+
+When creating a new release, there are some guidelines to follow before starting
+the process:
+
+ * Ask the team if there are any concerns about making the release.
+ * Check if there are changes to previous library versions that need to be forward-ported.
+ * See if there are open PRs against the release branch (release branch is the
+   one with the name `ign-fooX` where foo is the name of the Ignition library and
+   X the major version of the version bump planned) that could go into the new
+   release.
+
+#### Software and configurations
+
+The releasing process is only supported on Linux at this moment.
+
+The tool designed to facilitate the process of releasing software for all
+platforms is called [`release.py`](https://github.com/ignition-tooling/release-tools/blob/master/release.py), and is hosted at https://github.com/ignition-tooling/release-tools.
+Cloning the `release-tools` repository is required to perform a new release:
+
+```
+cd ~/
+git clone https://github.com/ignition-tooling/release-tools.git
+```
+
+The `release.py` tool will use `sc3md` software to interact with Amazon AWS
+storage to host sources and binaries, so make sure `sc3md` is installed:
+
+```
+sudo apt-get install s3cmd
+```
+
+Some Debian tools require the following variables to be set:
+```bash
+export DEBEMAIL="<username>@openrobotics.org"
+export DEBFULLNAME="<Your full name>"
+```
+**Note:** the two above exported variables can be added to `.bashrc` to have
+them configured automatically on every run.
+
+
+#### Credentials
+
+There are some credentials needed to interact with the release
+process:
+
+ * S3 access to Open Robotics:
+  ```
+  s3cmd --configure
+  ```
+  If you don't have AWS credentials, please contact @j-rivero or @nuclearsandwich who will help set you up.
+
+ * Release token: magic sequence of characters needed while running `release.py`
+   to interact with `build.osrfoundation.org`. This should be given to Ignition releasers as a part of the AWS credentials set-up.
+
+### Preparing Ignition Code
+
+## Update code version
+
+The first step to get a new release ready is to update the current code (upstream)
+version (view the [versioning](#versioning) section for more information). This
+bump could be in the major number (non-compatible changes), minor number (new
+features), patch number (patches and bugfixes).
+
+1. **Bumping major number** of the version implies some work to have the
+[metadata](#metadata-for-releasing) updated correctly. There is a [dedicated
+document](releasing/bump_major.md) that you should go through before continuing to work through the steps in this
```

Comment:
The 'dedicated document' link should work once the PR is merged, AFAIK. Path seems correct to me:
```bash
docs on releasing_stepbystep [?] ❯ find . -name bump_major*
./releasing/bump_major.md
```

Reply:
That's kinda weird, I don't see it in the branch :o 

https://github.com/gazebosim/docs/tree/releasing_stepbystep/releasing

Reply:
Also I think it'd be nice if this section was separated or better emphasized, because I missed it the first time around and only realized because I knew I had to run the changelog generation in the main README

Reply:
Sorry for the missing file, I've uploaded it in this PR since it was only present locally.

Reply:
> Also I think it'd be nice if this section was separated or better emphasized, because I missed it the first time around and only realized because I knew I had to run the changelog generation in the main release-tools README

I've cleanup the index and added STEP in there, hopefully is easier now a4c7b71869cf422909ff3dea70be42f23ec2a88d